index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require_once 'rif-collector-app.php';
+$db = returnDb();
+
+function retrieveAllQuery($db) {
+    $query = $db->prepare("SELECT `rifs`.`make`, `rifs`.`model`, `rifs`.`type`, `colors`.`color` AS 'color', `rifs`.`mags_owned`, `rifs`.`power_source`, `rifs`.`sites_visited`, `rifs`.`purchase_date` FROM `rifs` JOIN `colors` ON `rifs`.`color` = `colors`.`id`;");
+    $result = $query->execute();
+    return $query->fetchAll();
+}
```

Comment:
Why is this function definition in the index file when your other functions are happily in their own other file?

Reply:
Was previously directed to leave it there, **actioned: moved with others**.

Would like to separate all functions into separate files but at the same point previously was directed that it would be fine for now.